app/config/locales/en.yml

```
@@ -46,14 +46,18 @@ en:
         subheader: Your caseworker needs your payment information from the last 30 days.
     entries:
       show:
-        case_number: Case Number
-        credentials: Your log in credentials to your employer’s payroll provider account
-        employer_name: Your employer name or the name of your employer’s payroll provider
+        case_number: 'We’ll be verifying income related to this ID number: %{case_number}'
+        do_app_based_work: People who do app-based gig work (like Uber, Lyft, DoorDash)
         get_started: Get Started
+        has_online_payroll_provider: People with an employer who has an online payroll provider. Most people who receive pay stubs or W-2 tax forms have an online payroll provider. If you’re unsure, ask your employer.
         header: Let's verify your income through your payment information
-        list_header: 'Before you begin, make sure you have this information:'
         security_message: Your information is secure. We will not share or keep your log in information. You will be able to preview and approve everything that is shared with your caseworker.
         subheader: We’ll help you connect payment records from your employer to your caseworker so you can prove your income and receive benefits faster.
+        subheader_1: Who can use this tool
+        subheader_2: What you need
+        you_can_get_started: You can get started right away with this information
+        your_employers_name: Your employer’s name or the name of their payroll provider
```

Side note but I've been noticing that Figma seems to be copy-pasting smart quotes (in this string and the others as well). I don't imagine it will be a problem, but if we see any weird rendering issues with these text strings, that's probably why.

```suggestion
        your_employers_name: Your employer's name or the name of their payroll provider
```